src/sound_check_menu.c

```diff
@@ -0,0 +1,2198 @@
+    SEName_ELEBETA,
+    SEName_HINSI,
+    SEName_EXPMAX,
+    SEName_TAMAKORO,
+    SEName_TAMAKORO_E,
+    SEName_BASABASA,
+    SEName_REGI,
+    SEName_C_GAJI,
+    SEName_C_MAKU_U,
+    SEName_C_MAKU_D,
+    SEName_C_PASI,
+    SEName_C_SYU,
+    SEName_C_PIKON,
+    SEName_REAPOKE,
+    SEName_OP_BASYU,
+    SEName_BT_START,
+    SEName_DENDOU,
+    SEName_JIHANKI,
+    SEName_TAMA,
+    SEName_Z_SCROLL,
+    SEName_Z_PAGE,
+    SEName_PN_ON,
+    SEName_PN_OFF,
+    SEName_Z_SEARCH,
+    SEName_TAMAGO,
+    SEName_TB_START,
+    SEName_TB_KON,
+    SEName_TB_KARA,
+    SEName_BIDORO,
+    SEName_W085,
+    SEName_W085B,
+    SEName_W231,
+    SEName_W171,
+    SEName_W233,
+    SEName_W233B,
+    SEName_W145,
+    SEName_W145B,
+    SEName_W145C,
+    SEName_W240,
+    SEName_W015,
+    SEName_W081,
+    SEName_W081B,
+    SEName_W088,
+    SEName_W016,
+    SEName_W016B,
+    SEName_W003,
+    SEName_W104,
+    SEName_W013,
+    SEName_W196,
+    SEName_W086,
+    SEName_W004,
+    SEName_W025,
+    SEName_W025B,
+    SEName_W152,
+    SEName_W026,
+    SEName_W172,
+    SEName_W172B,
+    SEName_W053,
+    SEName_W007,
+    SEName_W092,
+    SEName_W221,
+    SEName_W221B,
+    SEName_W052,
+    SEName_W036,
+    SEName_W059,
+    SEName_W059B,
+    SEName_W010,
+    SEName_W011,
+    SEName_W017,
+    SEName_W019,
+    SEName_W028,
+    SEName_W013B,
+    SEName_W044,
+    SEName_W029,
+    SEName_W057,
+    SEName_W056,
+    SEName_W250,
+    SEName_W030,
+    SEName_W039,
+    SEName_W054,
+    SEName_W077,
+    SEName_W020,
+    SEName_W082,
+    SEName_W047,
+    SEName_W195,
+    SEName_W006,
+    SEName_W091,
+    SEName_W146,
+    SEName_W120,
+    SEName_W153,
+    SEName_W071B,
+    SEName_W071,
+    SEName_W103,
+    SEName_W062,
+    SEName_W062B,
+    SEName_W048,
+    SEName_W187,
+    SEName_W118,
+    SEName_W155,
+    SEName_W122,
+    SEName_W060,
+    SEName_W185,
+    SEName_W014,
+    SEName_W043,
+    SEName_W207,
+    SEName_W207B,
+    SEName_W215,
+    SEName_W109,
+    SEName_W173,
+    SEName_W280,
+    SEName_W202,
+    SEName_W060B,
+    SEName_W076,
+    SEName_W080,
+    SEName_W100,
+    SEName_W107,
+    SEName_W166,
+    SEName_W129,
+    SEName_W115,
+    SEName_W112,
+    SEName_W197,
+    SEName_W199,
+    SEName_W236,
+    SEName_W204,
+    SEName_W268,
+    SEName_W070,
+    SEName_W063,
+    SEName_W127,
+    SEName_W179,
+    SEName_W151,
+    SEName_W201,
+    SEName_W161,
+    SEName_W161B,
+    SEName_W227,
+    SEName_W227B,
+    SEName_W226,
+    SEName_W208,
+    SEName_W213,
+    SEName_W213B,
+    SEName_W234,
+    SEName_W260,
+    SEName_W328,
+    SEName_W320,
+    SEName_W255,
+    SEName_W291,
+    SEName_W089,
+    SEName_W239,
+    SEName_W230,
+    SEName_W281,
+    SEName_W327,
+    SEName_W287,
+    SEName_W257,
+    SEName_W253,
+    SEName_W258,
+    SEName_W322,
+    SEName_W298,
+    SEName_W287B,
+    SEName_W114,
+    SEName_W063B
+};
+
+void sub_80BA258(u8);
```

Comment:
Would be better to move the function declarations to before the data, to be consistent with other files that might reference functions in their data.

Thinking of:
- Includes
- Macros
- Constants
- Global variables
- Structs
- Function predeclarations
- Data
- Functions